polymesh_schema.json

```
@@ -27,12 +27,13 @@
                 "Custom": "Vec<u8>"
             }
         },
-        "IdentifierType": {
+        "Identifier": {
```

```suggestion
        "AssetIdentifier": {
```
Since the type was renamed?